docs/index.md

```diff
@@ -15,5 +15,23 @@ Read about the new features of the Onegini Identity Cloud in our platform update
 * [November 2020](https://blog.onegini.com/onegini-identity-cloud-platform-update-november-2020)
 * [August 2020](https://blog.onegini.com/onegini-identity-cloud-platform-update-august-2020)
 
-We are rewriting our documentation for the Onegini Identity Cloud. You can find the single-tenant documentation
-on [docs-single-tenant.onegini.com](https://docs-single-tenant.onegini.com/).
\ No newline at end of file
+## Which documentation should I use?
+
+We are rewriting our documentation for the multi-tenant Onegini Identity Cloud. Eventually you will find all information about the Onegini Identity Cloud on 
+this site.
+
+On this site you can find the latest multi-tenant documentation for:
```

Comment:
We have the `on this site` repeating in the sentences, maybe rephrase to:
```
Here you can find the latest multi-tenant documentation for:
```
or other?